classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+end
```

Comment:
Nice job so far😉. Just as required for this milestone, kindly add a method that serves as the 'has-many' side (a classroom has many students)😉. You can take a look at[ this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more clarity😉.

<img width="721" alt="image" src="https://github.com/zamcham/OOP-school-library/assets/64643995/af2c452a-2eba-487a-93e8-4bc64c0c096b">